authgear/templates/authgear-portal-server.yaml

```diff
@@ -342,10 +350,17 @@ metadata:
     {{- include "authgear.labels" . | nindent 4 }}
 spec:
   ports:
+{{- if (eq .Values.authgear.portalServer.authgear.webSDKSessionType "cookie") }}
   - port: 80
     targetPort: proxy
     protocol: TCP
     name: http
+{{- else }}
+  - port: 80
+    targetPort: portal
+    protocol: TCP
+    name: http
+{{- end }}
```

Comment:
I think access token should work with the proxy too so maybe we can simply always use the proxy?

Reply:
I will not include a separate proxy for the portal in ONCE, so it is better align the deployment. When sessionType=refresh_token, there is no dedicated proxy in front of the portal.

Reply:
staging has been updated to use this change now. Encountered an issue https://linear.app/authgear/issue/DEV-2432/validatescopes-does-not-respect-always-allowed-grant-types but that can be worked around by setting grant_types explicitly.

Reply:
This should be good to merge.